lock-impl/src/main/java/com/palantir/lock/impl/LockServiceImpl.java

```diff
@@ -471,20 +469,24 @@ private void tryLocks(LockClient client, LockRequest request, BlockingMode block
         }
     }
 
-    private void logSlowLockAcquisition(String lockId, LockClient currentHolder, long duration) {
-        if (slowLogEnabled) {
+    private void logSlowLockAcquisition(String lockId, LockClient currentHolder, long durationMillis) {
```

Comment:
There are a few forks and flows of control here. Is it time to extract some little testable helper for our slow logging?